src/styles/card-block.scss

```diff
@@ -0,0 +1,116 @@
+@import "./stars.scss";
+
+@import "./variables.scss";
+
+.container {
+  max-width: $card-width;
+
+  background: $white-color;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding-top: 32px;
+  padding-bottom: 16px;
+  margin-right: $card-margin-right;
+  margin-bottom: $card-margin-bottom;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    display: flex;
+  }
+
+  &__info {
+    padding: 0 16px;
+  }
+
+  &__heading {
+    margin-bottom: 4px;
+
+    display: flex;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $dark-blue;
+  }
+  &__text-small {
+    color: $grey-color;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+
+    font-size: 700;
+
+    margin-bottom: 16px;
+
+    &-text {
+      font-weight: 400;
+      font-size: 12px;
+      color: $grey-color;
+
+      display: flex;
+      align-items: flex-end;
+    }
```

Comment:
Don't use `&` to combine parts of class into one name, use it for BEM elements inside block (or modifiers), for pseudoclasses and pseudoelements, but not for that purpose